### PR TITLE
ignore response logs for paths in ignorePaths

### LINKED ISSUE #63: response logEvents should honor ignorePaths
It seems `ignorePaths` is only used when `logEvent` is `request` but would be beneficial when the log event is `response` as well. Like the **README** states, ignoring certain paths can reduce the noise in the logs for health check routes (Kubernetes pings a service quite frequently for health check routes).

If this sounds good I can submit a PR when time permits.

### PR DESCRIPTION
**Pull request by robmcguinness**
closes #63

### CODE REVIEW

**Comment by felixheck** [comment]
LGTM

File: index.js [hunk 1]
```diff
@@ -106,6 +106,14 @@ async function register (server, options) {
 
   // log when a request completes
   tryAddEvent(server, options, 'on', 'response', function (request) {
+    if (options.ignorePaths && ignoreTable[request.url.path]) {
+      return
+    }
+
+    if (!request.response) {
+      return
+    }
```
Comment (mcollina) [inline index.js]: Why is this check needed? code coverage is reporting that this is not covered by tests: https://coveralls.io/builds/18603822/source?filename=index.js.

**Comment by robmcguinness** [comment]
I'll fix the code coverage drop today

**Comment by robmcguinness** [comment]
does this look good to merge and release? let me know if more is needed. thanks!

**Comment by mcollina** [comment]
Awesome!

**Pull request merged**